test_module/test_path.py

```diff
@@ -0,0 +1,105 @@
+import json
+import pytest
+from project.utils import create_fake_dict, create_output_file, existing_dir, clear_files_in_path, construct_files
+import os
+
+@staticmethod
+def validateJsonText(jsonText):
+    try:
+        json.loads(jsonText)
+    except ValueError as err:
+        print(err)
+        return False
+    return True
+
+
+def test_dict_len_matches_key_number():
+    data_schema = "{\"date\": \"timestamp:\",\"name\": \"str:rand\",\"type\": \"['client', 'partner', 'government']\"," \
+                  "\"animal_type\": \"['cat', 'dog', 'monkey','tiger']\",\"age\": \"int:rand(1, 90)\",\"kids_number\": " \
+                  "\"int:rand(1, 6)\"} "
+    dict_given = create_fake_dict(data_schema)
+    dict_values = dict_given.values()
+    dict_keys = dict_given.keys()
+    assert len(dict_values) == len(dict_keys)
+
+def test_add_lines_to_file():
+    file_name = "magic_file_generator.json"
+    d_schema = "{\"date\": \"timestamp:\"} "
+    path = "/Users/kpodlaska/Desktop"
+    f_line_given = 10
+    file_given = create_output_file(f_line_given, d_schema, path, file_name)
+    print(file_given)
+    f = open(file_given)
+    data = json.load(f)
+    f.close()
+    assert f_line_given == len(data)
+
+@staticmethod
+def count_files_in_path(path, file_name):
+    existing_dir(path)
+    "check if path exists"
+    files = os.listdir(path)
+    result = []
+    for file in files:
+        if file.endswith("json"):
+            if file_name in file:
+                result.append(file)
+    return len(result)
+
+
+
+def test_creating_number_of_files():
+    expected_value = 1
+    d_schema = "{\"date\": \"timestamp:\"} "
+    f_name = "test_file_for_testing_path.json"
+    path= "/Users/kpodlaska/Desktop"
+    cleaning_before_testing = clear_files_in_path(path, f_name)
+    create_output_file(20, d_schema, path, f_name)
+    assert count_files_in_path(path, f_name) == expected_value
+
+
+
+def test_clear_path():
+        how_many_files = 5
+        f_line = 1
+        d_schema = "{\"date\": \"timestamp:\"} "
+        f_name = "test_file_for_testing_path.json"
+        prefix="count"
+        path = "/Users/kpodlaska/Desktop"
```

Comment:
magic string. To run all these tests I need to replace this path by my own value manually:(